test/Swashbuckle.AspNetCore.SwaggerGen.Test/XmlComments/XmlCommentsDocumentFilterTests.cs

```diff
@@ -155,6 +154,25 @@ public void Ensure_IncludeXmlComments_Adds_Filter_To_Options()
             Assert.Contains(options.DocumentFilters, x => x is XmlCommentsDocumentFilter);
         }
 
+        [Fact]
+        public void Ensure_IncludeXmlCommentsForAssembly_Adds_Filter_To_Options()
+        {
+            var services = new ServiceCollection();
+            services.AddSingleton<IWebHostEnvironment, DummyHostEnvironment>();
+            services.AddSwaggerGen(c =>
+            {
+                c.IncludeXmlCommentsForAssembly(
+                    typeof(FakeControllerWithXmlComments).Assembly,
+                    includeControllerXmlComments: true);
```

Comment:
This makes me wonder if you could make this _even more_ convenient with an overload such as:

```csharp
// Not sure what the best name would be
public static void IncludeXmlComments<T>(
    this SwaggerGenOptions swaggerGenOptions,
    bool includeControllerXmlComments = false)
{
    var assembly = typeof(T).Assembly;
    return swaggerGenOptions.IncludeXmlCommentsForAssembly(assembly, includeControllerXmlComments);
}

// Usage
c.IncludeXmlComments<FakeControllerWithXmlComments>(includeControllerXmlComments: true);
```

Reply:
I think this is incorrect layering. We are operating in assembly terms, and while we are using class for locating assembly in our code, we don't actually perform any action on class. I prefer it be explicit in caller's code.

I'm investigated providing `Assembly.GetCallingAssembly()` overload, but it seems to be unstable

May be it worth to provide method that uses `GetEntryAssembly()`

Reply:
The entry assembly is almost always wrong in tests, so I don't think we should introduce it.